apps/main/src/layout/default/index.tsx

```diff
@@ -72,6 +72,7 @@ const getSections = (locale: Locale, network: string) => [
       { route: 'https://news.curve.fi/', label: t`News` },
       { route: 'https://resources.curve.fi/lending/understanding-lending/', label: t`User Resources` },
       { route: 'https://docs.curve.fi', label: t`Developer Resources` },
+      { route: `${network ? `/${network}` : ''}${ROUTE.PAGE_DISCLAIMER}`, label: t`Risk Disclaimers` },
```

Comment:
unrelated to this PR, but this is so ugly :cry:

Reply:
Do you think using a query parameter for network selection would be better, as it's more a thing about the app state than it is about an actual route? Not that it would completely solve the ugliness, but it might be a tad bit better and prevent some routing problems we have currently.

Reply:
Well if it's optional it should indeed be in the query or in the end of the route 🤔